docs/src/main/sphinx/functions/string.rst

```diff
@@ -100,6 +100,19 @@ String functions
     Tests whether a ``string`` of digits is valid according to the
     `Luhn algorithm <https://en.wikipedia.org/wiki/Luhn_algorithm>`_.
 
+    This checksum function also known as "modulo 10" or "mod 10" is
+    widely applied on most credit card numbers as well as government identification numbers
```

Comment:
widely applied on credit card numbers and government identification numbers to distinguish valid numbers from mistyped, incorrect numbers.

Reply:
the suggestion was not applied yet .. any specific reason?

Reply:
I have misread your suggestion and applied only a part of it. Sorry for the misunderstanding.
I have changed now the sentence.